react/features/base/config/reducer.js

```diff
@@ -194,6 +194,26 @@ function _translateLegacyConfig(oldValue: Object) {
         newValue.toolbarButtons = interfaceConfig.TOOLBAR_BUTTONS;
     }
 
+    if (oldValue.disableJoinLeaveSounds) {
+        newValue.disabledSoundIds = (newValue.disabledSoundIds || [])
```

Comment:
Maybe we can just make sure it's an array before we enter here to avoid having to do `(newValue.disabledSoundIds || [])` 3 times?